scarb/src/compiler/plugin/proc_macro/compilation.rs

```diff
@@ -64,6 +65,10 @@ pub fn fetch_package(package: &Package, ws: &Workspace<'_>) -> Result<()> {
     run_cargo(CargoAction::Fetch, package, ws)
 }
 
+pub fn package_package(package: &Package, opts: &PackageOpts, ws: &Workspace<'_>) -> Result<()> {
```

Comment:
The intention was to align with `fetch_package` above, and with the fact that just `package` might not be descriptive enough in this particular case.

Maybe this would be better, since we're dealing with cargo anyway?
```suggestion
pub fn package_crate(package: &Package, opts: &PackageOpts, ws: &Workspace<'_>) -> Result<()> {
```

Would be reasonable to rename `fetch_package` then as well though.

Reply:
> The intention was to align with fetch_package

Yep, I understand. But still sounds strange :D 

> package_crate

👍

Reply:
Addressed in [8464751](https://github.com/software-mansion/scarb/pull/1605/commits/8464751fcc142eef312f14257eefe4508dd3ddf0)